site2/docs/reference-metrics.md

```diff
@@ -605,16 +605,19 @@ All the web service executor metrics are labeled with the following labels:
 
 ### Metadata store metrics
 
-All the metadata store metrics are labelled with the following labels:
+All the metadata store metrics are labeled with the following labels:
 
 - *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you configured in `broker.conf`.
 - *name*: `name=${metadata-store|configuration-metadata-store|state-metadata-store}`. `${name}` is the metadata store name.
 
-| Name                                   | Type      | Description                                                                                  |
-|----------------------------------------|-----------|----------------------------------------------------------------------------------------------|
-| pulsar_metadata_store_ops_latency      | Histogram | The latency of getting/deleting/putting data fail/success operations from/to metadata store. |
-| pulsar_metadata_store_put_bytes_total  | Counter   | The number of data put to metadata store.                                                    |
-
+| Name                                               | Type      | Description                                                                                  |
+|----------------------------------------------------|-----------|----------------------------------------------------------------------------------------------|
+| pulsar_metadata_store_ops_latency                  | Histogram | The latency of getting/deleting/putting data fail/success operations from/to metadata store. |
```

Comment:
Sorry, what does `The latency of getting/deleting/putting data fail/success operations from/to metadata store.` this sentence mean?

Reply:
Thanks! Can we change it to `The latency of getting/deleting/putting data from/to metadata store.` (delete `fail/success operations` since it brings confusion?